backend/src/db/migrations/20251211120000_pam-ssh-resource-ca-keypair.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  if (!(await knex.schema.hasColumn(TableName.PamResource, "encryptedCaPrivateKey"))) {
+    await knex.schema.alterTable(TableName.PamResource, (t) => {
+      t.binary("encryptedCaPrivateKey").nullable();
+    });
+  }
+  if (!(await knex.schema.hasColumn(TableName.PamResource, "caPublicKey"))) {
+    await knex.schema.alterTable(TableName.PamResource, (t) => {
+      t.text("caPublicKey").nullable();
+    });
+  }
+  if (!(await knex.schema.hasColumn(TableName.PamResource, "caKeyAlgorithm"))) {
+    await knex.schema.alterTable(TableName.PamResource, (t) => {
+      t.string("caKeyAlgorithm").nullable();
+    });
+  }
```

Comment:
hmmm maybe we should just have one encrypted column to store these resource-type-specific values? Else, PAM resource would have so many nullable columns in the future